modules/core/exported/client.go

```diff
@@ -21,6 +21,15 @@ const (
 	// Localhost is the client type for a localhost client. It is also used as the clientID
 	// for the localhost client.
 	Localhost string = "09-localhost"
+
+	// Active is a status type of a client. An active client is allowed to be used.
+	Active string = "Active"
+
+	// Frozen is a status type of a client. A frozen client is not allowed to be used.
+	Frozen string = "Frozen"
+
+	// Unknown indicates there was an error in determining the status of a client
+	Unknown string = "Unknown"
```

Comment:
should these be proto enums?

Reply:
No? They aren't used in communication. So I think it is probably simplest to leave them as strings. (only use proto when necessary)